### ISSUE TITLE
Exception if the `from` time is in the future 

### ISSUE THREAD

**Issue by sphh** [guludo/timewarrior-balance#5]
I just installed `balance` (globally) and generated the following `balance.conf` file:
```yaml
# Hours for the "work" tag
work {
  # Periodic configuration
  from 2023-10-01 {
    tue   8:00
    wed   8:00
    thu   3:30
  }
}
```

When running `timew bal` I get the following exception:
```python3
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.10/dist-packages/timewarrior_balance/__main__.py", line 3, in <module>
    balance.main()
  File "/usr/local/lib/python3.10/dist-packages/timewarrior_balance/balance.py", line 126, in main
    widths = tuple(
  File "/usr/local/lib/python3.10/dist-packages/timewarrior_balance/balance.py", line 127, in <genexpr>
    max(len(header[i]), len(total_row[i]), *(len(r[i]) for r in rows))
TypeError: 'NoneType' object is not subscriptable
'/home/sphh/.timewarrior/extensions/balance' returned 1 without producing output.
```

**Comment by guludo** [comment]
@sphh, thanks for reporting this.

I think I was able to reproduce the bug here. It seems to happen when the report internal does not match any data or configuration entry.

For example, that would happen if you have a fresh timewarrior database and run `timew bal` **today** (2023-08-23) using the shared balance.conf, which has configuration defined for the future. 

I have just pushed [674df30](https://github.com/guludo/timewarrior-balance/commit/674df3083bab5d63b62183ee229e349de0efc02b) as a fix for this. Does that fix the issue?
